les/client_handler.go

```diff
@@ -114,11 +114,25 @@ func (h *clientHandler) handle(p *serverPeer) error {
 		p.Log().Debug("Light Ethereum handshake failed", "err", err)
 		return err
 	}
+	// Register peer with the server pool
+	if h.backend.serverPool != nil {
```

Comment:
Is there any possibility that the serverpool can be nil? Perhaps in the testing?